openverse_catalog/dags/data_refresh/dag_factory.py

```diff
@@ -64,6 +66,15 @@
     f"{REFRESH_POPULARITY_METRICS_GROUP_ID}"
     f".{UPDATE_MEDIA_POPULARITY_METRICS_TASK_ID}"
 )
+API_DB_CONN_ID = "postgres_openledger_api"
```

Comment:
Should we get this from the `OPENLEDGER_API_CONN_ID` environment variable, and maybe add it to https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/common/constants.py?

Reply:
Fully support adding this to constants!! That's a great idea.